docs/learn/learn-inflation.md

```diff
@@ -11,19 +11,20 @@ import RPC from "./../../components/RPC-Connection";
 
 {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} is an inflationary token. On the
 {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} network, inflation is
-{{ polkadot: [set to be 10% annually](https://github.com/paritytech/polkadot-sdk/blob/756ccc35e93d1a78e3c71a0e67ae4da5f1d09f69/runtime/polkadot/src/lib.rs#L576), :polkadot }}
-{{ kusama: [set to be 10% annually](https://github.com/paritytech/polkadot-sdk/blob/756ccc35e93d1a78e3c71a0e67ae4da5f1d09f69/runtime/kusama/src/lib.rs#L535), :kusama }}
-which is distributed as staking rewards based on the amount staked vs. ideal staking rate, and the
-remainder goes to the treasury.
+{{ polkadot: [set to be 10% annually](https://github.com/paritytech/polkadot-sdk/blob/756ccc35e93d1a78e3c71a0e67ae4da5f1d09f69/runtime/polkadot/src/lib.rs#L576). :polkadot }}
+{{ kusama: [set to be 10% annually](https://github.com/paritytech/polkadot-sdk/blob/756ccc35e93d1a78e3c71a0e67ae4da5f1d09f69/runtime/kusama/src/lib.rs#L535). :kusama }}Depending
+on the {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} supply locked and the ideal staking
+rate (more about this below), part of the {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }}
+inflation goes to stakers and part to the [treasury](./learn-polkadot-opengov-treasury.md).
 
 :::info
 
 DOT went through [redenomination](./learn-redenomination.md) in 2020 that saw the DOT token supply
 increase by 100 times.
 
 The current token supply on {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} is
-{{ polkadot: <RPC network="polkadot" path="query.balances.totalIssuance" defaultValue={12230666300429914781} filter="humanReadable"/> (Over 1.2 Billion DOT). :polkadot }}
-{{ kusama: <RPC network="kusama" path="query.balances.totalIssuance" defaultValue={12619256191792480093} filter="humanReadable"/> (Over 12 Million KSM). :kusama }}
```

Comment:
<img width="811" alt="image" src="https://github.com/w3f/polkadot-wiki/assets/86818441/eaa94dba-e8fe-49b8-ae72-e1ff23278b69">

Knowing the quantity of DOT or KSM by taking a glance at this number is tough (especially for those who are not from the ecosystem and who do not have any idea about the total issuance. That is why an approximate human readable number quantifying the supply was added. If you plan on making this accessible for diverse audience, either automate this in the human readable filter or keep the approximate values as is.

Reply:
re-added, will create a filter in another PR